python/pyspark/sql/tests/test_dataframe.py

```diff
@@ -114,7 +114,6 @@ def test_count_star(self):
 
         self.assertEqual(df3.select(count(df3["*"])).columns, ["count(1)"])
         self.assertEqual(df3.select(count(col("*"))).columns, ["count(1)"])
-        self.assertEqual(df3.select(count(col("s.*"))).columns, ["count(1)"])
```

Comment:
Why did we remove this?

Reply:
Because it is wrong. Or at least inconsistent with the the sql frontend.